firstvoices/backend/resources/dictionary.py

```diff
@@ -119,3 +139,32 @@ class DictionaryEntryRelatedCharacterResource(BaseResource):
 
     class Meta:
         model = DictionaryEntryRelatedCharacter
+
+
+class DictionaryEntryLinkResource(BaseResource):
+    from_dictionary_entry = fields.Field(
+        column_name="dictionary_entry",
+        attribute="from_dictionary_entry",
+        widget=ForeignKeyWidget(DictionaryEntry, "id"),
+    )
+
+    to_dictionary_entry = fields.Field(
+        column_name="related_entry",
+        attribute="to_dictionary_entry",
+        widget=ForeignKeyWidget(DictionaryEntry, "id"),
+    )
+
+    class Meta:
+        model = DictionaryEntryLink
+
+    def before_import_row(self, row, row_number=None, **kwargs):
+        # skip rows with non-existent "to" and "from" dictionary entries
+        logger = logging.getLogger(__name__)
+        try:
+            DictionaryEntry.objects.get(id=row["related_entry"])
+        except DictionaryEntry.DoesNotExist:
+            logger.warning(
+                f"Skipping row {row_number} because entry {row['related_entry']} does not exist."
+            )
+            row["related_entry"] = ""
+            # TODO: prevent the row from being imported
```

Comment:
Same as above comment: I could not quite figure out how to skip importing the row, if someone wants to pick this ticket up this is the only work left to do. Thanks!